README.md

```diff
@@ -19,7 +19,7 @@ Installation
 Example Usage for Bittrex API
 
 ```python
-from bittrex import Bittrex
```

Comment:
Does `from bittrex import Bittrex` not work? I thought the import in the `__init__.py` would make it work. Seems like a nicer way to import versus `from bittrex.bittrex import Bittrex`